portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase

```diff
@@ -1095,6 +1095,76 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
 			userEmailAddress = "test@liferay.com");
 	}
 
+	@description = "This is a test for LPS-129944. Out Of Memory Error when web content display widget's configuration is opened."
+	@priority = "4"
+	test ViewConfigurationInWCDIsSavedWithoutOOME {
+		task ("Add Web Content Structure") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			NavItem.gotoStructures();
+
+			WebContentStructures.addCP(structureName = "WC Structure Name");
+
+			DataEngine.addField(
+				fieldFieldLabel = "Text",
+				fieldName = "Text");
+
+			WebContentStructures.saveCP(structureName = "WC Structure Name");
+		}
+
+		task ("Add Web Content Template based on the Structure") {
+			NavItem.gotoTemplates();
+
+			LexiconEntry.gotoAdd();
+
+			WebContentTemplates.inputTemplateAddScreenFields(
+				structureName = "WC Structure Name",
+				templateDescription = "WC Template Description",
+				templateName = "WC Template Name");
+
+			var templateScript = '''<@liferay_portlet["runtime"] portletName="com_liferay_journal_content_web_portlet_JournalContentPortlet" instanceId="randomString" />''';
+
+			WebContentTemplates.inputScript(templateScript = "${templateScript}");
+
+			PortletEntry.save();
+		}
+
+		task ("Add Web Content based on the Structure") {
+			NavItem.gotoWebContent();
+
+			WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+
+			WebContent.addWithStructureCP(
+				structureName = "WC Structure Name",
+				templateName = "WC Template Name",
+				webContentTitle = "WC WebContent Title");
+
+			PortletEntry.publish();
+		}
+
+		task ("Select the Web Content and Check related assets on WCD configuration") {
+			Navigator.gotoSitePage(
+				pageName = "Web Content Display Page",
+				siteName = "Test Site Name");
+
+			WebContentDisplayPortlet.selectWebContent(webContentTitle = "WC WebContent Title");
```

Comment:
Wrong step

Reply:
Hi Tim, I didn't find other macro to select the web content, may you help me with this one?

Reply:
This task is not select a web content in WCD but add a web content to page via Content tab in Add panel.
![Expected](https://user-images.githubusercontent.com/52661397/122140543-1fd22900-ce7e-11eb-997e-044ef93ee73e.PNG)


Btw, I recommend create a new widget page then add web content to page.